### PR TITLE
Implement rush-select

### PR DESCRIPTION
**Pull request by idahogurl**
This [rush-select](https://github.com/TeliaSweden/rush-select) tool was recommended on the Rush website https://rushjs.io/pages/advanced/watch_mode/
The other tool recommended [rush-dev-watcher](https://github.com/dimfeld/rush-dev-watcher) requires us to include it in our repository as a separate project. 

Running `rush select` will give you an interface where you can choose what script to run on each project. Choose `dev` and all the projects will run in watch mode.

### CODE REVIEW

File: apps/hl/package.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "version": "0.1.0",
   "scripts": {
     "build": "next build",
-    "build:watch": "",
```
Comment (idahogurl) [inline apps/hl/package.json:7]: Not needed with `rush-select`

File: .prettierignore [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Keep this section in sync with .gitignore
 #-------------------------------------------------------------------------------------------------------------------
 
-# Logs
+## Logs
```
Comment (idahogurl) [inline .prettierignore:5]: Have `.prettierignore` match `.gitignore`

File: libs/components/package.json [hunk 3]
```diff
@@ -6,13 +6,13 @@
   "author": "Rebecca Vest",
   "main": "dist/main.js",
   "scripts": {
-    "build": "heft test --clean",
```
Comment (idahogurl) [inline libs/components/package.json:9]: Without being able to use Heft on NextJs projects, I had to put `lint` and `test` into the githooks.  No reason for Heft to run those in addition to the githooks.

**Review by hacknightly — APPROVED** [review]
Awesome 🎖️

**Pull request merged**